src/ert/scheduler/scheduler.py

```diff
@@ -124,18 +125,29 @@ async def run_done_callback(self, state: RealizationState) -> Optional[LoadStatu
             f"Running done callback for {state.realization.run_arg.iens} ..(partly blocking)..",
             end="",
         )
-        callback_status, state._callback_status_msg = await forward_model_ok(
-            state.realization.run_arg
-        )
+        callback_status, status_msg = await forward_model_ok(state.realization.run_arg)
         print(" done")
+
+        if state._callback_status_msg != "":
```

Comment:
Isn't this flipped?

Reply:
This looks strange but is copied directly from the existing job_queue_node, and it must be this way to pass tests ;)